LykkeApi2/Controllers/HistoryController.cs

```diff
@@ -24,34 +23,31 @@ public class HistoryController : Controller
         private readonly IOperationsHistoryClient _operationsHistoryClient;
         private readonly IRequestContext _requestContext;
         private readonly IClientAccountClient _clientAccountService;
-        private readonly DomainModelConverter _converter;
 
         public HistoryController(
             IOperationsHistoryClient operationsHistoryClient, 
             IRequestContext requestContext, 
-            IClientAccountClient clientAccountService, 
-            DomainModelConverter converter)
+            IClientAccountClient clientAccountService)
         {
             _operationsHistoryClient = operationsHistoryClient ?? throw new ArgumentNullException(nameof(operationsHistoryClient));
             _requestContext = requestContext ?? throw new ArgumentNullException(nameof(requestContext));
             _clientAccountService = clientAccountService ?? throw new ArgumentNullException(nameof(clientAccountService));
-            _converter = converter ?? throw new ArgumentNullException(nameof(converter));
         }
 
         /// <summary>
         /// Get history by client identifier
         /// </summary>
-        /// <param name="operationType">The type of the operation, possible values: CashInOut, CashOutAttempt, ClientTrade, TransferEvent, LimitTradeEvent</param>
+        /// <param name="operationType">The type of the operation, possible values: CashIn, CashOut, Trade</param>
         /// <param name="assetId">Asset identifier</param>
         /// <param name="take">How many maximum items have to be returned</param>
         /// <param name="skip">How many items skip before returning</param>
         /// <returns></returns>
         [HttpGet("client")]
         [SwaggerOperation("GetByClientId")]
         [ProducesResponseType(typeof(ErrorResponse), (int) HttpStatusCode.InternalServerError)]
-        [ProducesResponseType(typeof(IEnumerable<ApiHistoryOperation>), (int) HttpStatusCode.OK)]
+        [ProducesResponseType(typeof(IEnumerable<HistoryOperation>), (int) HttpStatusCode.OK)]
```

Comment:
Is HistoryOperation an internal service contract? If so, I don;t think it's a good idea to share internal contracts via public API